src/app/settings/components/SettingsTable/SettingsTable.test.tsx

```diff
@@ -54,35 +64,39 @@ describe("SettingsTable", () => {
         ]}
         loaded={false}
         loading={true}
-      />
+      />,
+      {}
     );
 
-    expect(wrapper.find(".p-table-actions__space-left").exists()).toBe(true);
-    expect(wrapper.find("SearchBox").exists()).toBe(false);
+    expect(screen.queryByText("Search")).not.toBeInTheDocument();
   });
 });
 
 it("can render a disabled button ", () => {
-  const wrapper = shallow(
+  renderWithBrowserRouter(
     <SettingsTable
       buttons={[{ label: "Add User", url: "/foo", disabled: true }]}
       loaded={false}
       loading={true}
-    />
+    />,
+    {}
   );
 
-  expect(wrapper.find("Button").props().disabled).toBe(true);
+  const button = screen.getByRole("link", { name: "Add User" });
+  expect(button).toBeInTheDocument();
+  expect(button.classList.contains("is-disabled")).toBe(true);
```

Comment:
It's better to test for the disabled attribute (which affects how users can(not) interact with it), `is-disabled`. `className` can be useful to test, but it's more of an implementation detail (as `.is-disabled` className does not guarantee the proper style will be even applied, `testing-library`/`jsdom` doesn't load `css`).

```suggestion
  expect(button).toBeDisabled();
```

Reply:
I tried this, but the test appears to fail when using `.toBeDisabled()`, even though it has the `is-disabled` class:

```
Received element is not disabled:
      <a aria-disabled="true" class="p-button is-disabled" href="/foo" />
```